frontend/src/app/shared/components/storages/storage/storage.component.ts

```diff
@@ -201,9 +201,8 @@ export class StorageComponent extends UntilDestroyedMixin implements OnInit {
   }
 
   private instantiateStorageInformation(fileLinks:IFileLink[]):StorageInformationBox[] {
-    // ToDo: Replace with anonymous user capabilities check.
-    // https://community.openproject.org/projects/openproject/work_packages/44850/activity
-    if (!this.isLoggedIn) {
+    const projectId = (this.resource.project as unknown&{ id:string }).id;
+    if (!this.currentUserService.hasCapabilities$('file_links/view', projectId).pipe(this.untilDestroyed()).subscribe()) {
```

Comment:
We made a little logic mistake. We still need to go with the `isLoggedIn`. This is because for the anonymous user the info box shall never be visible. If we do it like that, it is also hidden from a logged-in user. It would be nice if you can revert this change.